File: Codeforces/A_Goals_of_Victory.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
 int t;
  cin>>t;
    while (t--)
    {
        int n; cin>>n;
        long long sum = 0;
        int arr[n];
        for(int i =0; i<n-1; i++){
            cin>>arr[i];
            sum += arr[i];
        }
        cout<<sum*-1<<endl;
    }
    return 0;
}

File: Codeforces/A_Goals_of_Victory.java
import java.util.Scanner;

public class A_Goals_of_Victory {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            int n = sc.nextInt();
            long sum = 0;
            int[] arr = new int[n - 1];

            for (int i = 0; i < n - 1; i++) {
                arr[i] = sc.nextInt();
                sum += arr[i];
            }

            System.out.println(sum * -1);
        }

        sc.close();
    }
}
